lib/text_handler.rb

```diff
@@ -0,0 +1,35 @@
+require 'dotenv'
+Dotenv.load('./twilio_credentials.env')
+require 'twilio-ruby'
+
+class TextHandler
+  def initialize
+    read_env_variables
```

Comment:
Since there are only three I would prefer to see the instance variables in `initialize`, where I expect to find them.